resources/metadata/cnbBuild.yaml

```
@@ -1,7 +1,9 @@
 metadata:
   name: cnbBuild
-  description: Executes a Cloud Native Buildpacks build for creating a Docker container.
-  longDescription: Executes a Cloud Native Buildpacks build for creating a Docker container.
+  description: Executes Cloud Native Buildpacks.
+  longDescription: |-
+    Executes a Cloud Native Buildpacks build for creating Docker container/s.
```

```suggestion
    Executes a Cloud Native Buildpacks build for creating Docker container(s).
```

---

Maybe OT, but it does not create docker containers, but docker images.
```suggestion
    Executes a Cloud Native Buildpacks build for creating Docker image(s).
```